### PR TITLE
Renaming WorkObject to Work

### PR DESCRIPTION
**Pull request by awead**
After our meeting last week, work objects are now called works. This relates to #348 because I think having `Work` as a module namespace makes sense for classes such as `Work::Deposit` and `Work::Type`

### CODE REVIEW

File: app/cho/work/change_set.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 
-module WorkObject
+module Work
   class ChangeSet < Valkyrie::ChangeSet
-    # self.fields = WorkObject::WorkObject.fields
+    # self.fields = Work::Work.fields
```
Comment (cam156) [inline app/cho/work/change_set.rb]: @awead I know this is not your comment, but this does make sense.  Any chance you know what it was supposed to be or maybe we should delete it.

File: app/cho/work/deposits_controller.rb [hunk 2]
```diff
@@ -54,17 +54,12 @@ def validate_save_and_render(change_set, error_view)
         end
       end
 
-      # Never trust parameters from the scary internet, only allow the white list through.
```
Comment (cam156) [inline app/cho/work/deposits_controller.rb]: 👍    I was looking at deleting this, but could not figure out why it was there in the first place.

File: app/cho/work/change_set.rb (hunk 1 above)
Comment (awead) [reply app/cho/work/change_set.rb]: Yeah, that's definitely not right... It should be Work::Deposit.

File: app/cho/work/deposits_controller.rb (hunk 2 above)
Comment (awead) [reply app/cho/work/deposits_controller.rb]: @cam156 I think it was part of the old generated controller code.

**Comment by awead** [comment]
@cam156 yes, I think it's a good idea to rename deposit as well. Submission makes the most sense to me right now.

**Pull request merged**